src/lexical_linter.coffee

```diff
@@ -18,10 +18,10 @@ BaseLinter = require './base_linter.coffee'
 #
 module.exports = class LexicalLinter extends BaseLinter
 
-    constructor: (source, config, rules, CoffeeScript) ->
+    constructor: (source, config, rules, CoffeeScript, tokens) ->
```

Comment:
This is so we can pass in the tokens from coffeelint.coffee (so we're not running `CoffeeScript.tokens(source)` twice)